app/src/main/AndroidManifest.xml

```diff
@@ -138,6 +138,7 @@
     <activity
       android:name=".app.profile.AdminPinActivity"
       android:label="@string/admin_pin_activity_title"
+    android:theme="@style/OppiaThemeWithoutActionBar"
```

Comment:
nit: this looks like it is not properly formatted. can you do `cmd+alt+L` to format your code?

Reply:
can i just edit it here !? or need to make commit through local.

Reply:
It's best to make your commits locally and push.